### DISCUSSION TITLE
Is there any way to covert property name to db attribute name?

### DISCUSSION THREAD

**Discussion by mrceperka** [prisma/prisma#2651] [Q&A] [answered]
Hello,

please consider this example prisma schema.
```prisma
model User {
  id                   String                 @default(cuid()) @id
  createdAt            DateTime               @default(now()) @map("created_at")
}
```

Now, let's say I have to do a `raw` query with `createdAt` field, but I do not want to type `created_at` or remember the mapping.

So I was hoping, that prisma could convert that for me.
Some method like :point_down: would be awesome.
 ```js 
const nameOfThePropertyInDatabase = prisma.user.propertyToStorageKey('createdAt')`
// nameOfThePropertyInDatabase = created_at
```

Could this API be exposed/created?

Thanks :)

**Comment by ryands17** [comment, marked as answer]
Hey @mrceperka :wave: 
Currently there's no such feature in Prisma, but could you create a Feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) stating your use case.

**Reply by mrceperka** [reply]
Sure thing, will do :) Thanks

**Reply by mrceperka** [reply]
https://github.com/prisma/prisma/issues/2652

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
